coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -292,6 +292,46 @@ def matmul(context, node):
     res = mb.matmul(x=inputs[0], y=inputs[1], name=node.name)
     context.add(res)
 
+@register_torch_op
```

Comment:
Could you also add unit tests (implement `class TestEinsum:`) in [test_torch_ops.py](https://github.com/apple/coremltools/blob/master/coremltools/converters/mil/frontend/torch/test/test_torch_ops.py)?

Using a few examples from https://pytorch.org/docs/master/generated/torch.einsum.html would be great.

Something like this:

```
class TestEinsum:

    @pytest.mark.parametrize("backend, equation", backends, ["abcd,cde->abe", ...])
    def test(self, backend, equation):

        if equation == "...":
            shape_a, shape_b = (3,4,5), (3,5,6)
        elif equation == "...":
            ...
        else:
            ...
        model = ModuleWrapper(function=torch.einsum)
        run_compare_torch(
            [shape_a, shape_b], model, backend=backend,
        )
```